protocol-units/da/m1/light-node/src/v1/sequencer.rs

```diff
@@ -420,31 +475,28 @@ impl LightNodeService for LightNodeV1 {
 
 		Ok(tonic::Response::new(grpc::BatchWriteResponse { blobs: intents }))
 	}
-	/// Update and manage verification parameters.
-	async fn update_verification_parameters(
-		&self,
-		request: tonic::Request<grpc::UpdateVerificationParametersRequest>,
-	) -> std::result::Result<
-		tonic::Response<grpc::UpdateVerificationParametersResponse>,
-		tonic::Status,
-	> {
-		self.pass_through.update_verification_parameters(request).await
-	}
 }
 
-mod block {
+pub mod block {
```

Comment:
Forgot about this one. It could be moved to a separate file as well, especially now that it's a public module (in its super module at least).